serving/docs/lmi/tutorials/tnx_aot_tutorial.md

```diff
@@ -0,0 +1,140 @@
+
+### Step 2: Pull the docker image
+
+Refer [here](https://github.com/aws/deep-learning-containers/blob/master/available_images.md#large-model-inference-containers) for the latest LMI NeuronX DLC and pull the image.
+
+For example:
+
+
+```
+# Login
+aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin 763104351884.dkr.ecr.us-east-1.amazonaws.com
+
+# Download
+docker pull 763104351884.dkr.ecr.us-east-1.amazonaws.com/djl-inference:0.26.0-neuronx-sdk2.16.0
+
+```
+
+### Step 3: Set the environment variables:
+
+These below configurations helps you configure the inference optimizations parameters. You can check all the configurations of LMI  NeuronX handler [in our docs](https://docs.djl.ai/docs/serving/serving/docs/lmi/configurations_large_model_inference_containers.html#transformers-neuronx-doc).
+
+
+
+```
+OPTION_MODEL_ID={{s3url}}
+OPTION_ENTRYPOINT=djl_python.transformers_neuronx
+OPTION_TENSOR_PARALLEL_DEGREE=24
+OPTION_ROLLING_BATCH=auto
+OPTION_MAX_ROLLING_BATCH_SIZE=16
+OPTION_DTYPE=fp16
+OPTION_N_POSITIONS=4096
+OPTION_ENABLE_MIXED_PRECISION_ACCUMULATION=true
+OPTION_MODEL_LOADING_TIMEOUT=12000
+OPTION_SAVE_MP_CHECKPOINT_PATH=/opt/ml/input/data/training/partition-test
+```
+
+You can also use serving.properties instead of these above environment variables (Preferred).
+
+
+```
+%%writefile serving.properties
+option.model_id={{s3url}}
+option.entryPoint=djl_python.transformers_neuronx
+option.tensor_parallel_degree=24
+option.rolling_batch=auto
+option.max_rolling_batch_size=16
+option.dtype=fp16
+option.n_positions=4096
+option.enable_mixed_precision_accumulation=true
+option.model_loading_timeout=12000
+option.save_mp_checkpoint_path=/opt/ml/input/data/training/partition-test
+```
+
+### Step 4: Run the ahead-of-time compilation
+
+Run the container and run the model partitioning script. 
+
+The command below assumes that we are using `serving.properties` and that we will be downloading the model artifacts from s3. If you need to download the model from HuggingFace using an access token you would add the following line
+
+```
+  ` -e HUGGING_FACE_HUB_TOKEN="hf_TOKEN_ENV_VALUE" \`
+```
+
+Similarly, if you want to use environmental variables rather than a `serving.properties` for defining your models configuration you would add them in the same way as the HuggingFace token.
+
+```
+docker run -t --rm --network=host \
+  -v /home/ubuntu/:/opt/ml/input/data/training \
+  --device /dev/neuron0 \
+  --device /dev/neuron1 \
+  --device /dev/neuron2 \
+  --device /dev/neuron3 \
+  --device /dev/neuron4 \
+  --device /dev/neuron5 \
+  --device /dev/neuron6 \
+  --device /dev/neuron7 \
+  --device /dev/neuron8 \
+  --device /dev/neuron9 \
+  --device /dev/neuron10 \
+  --device /dev/neuron11 \
+  763104351884.dkr.ecr.us-west-2.amazonaws.com/djl-inference:0.26.0-neuronx-sdk2.16.0 \
+  partition --model-dir /opt/ml/input/data/training --skip-copy
```

Comment:
missing model preparation steps, where do we get llama 70b and where do we place it

Reply:
Added